nym-vpn-cli/src/main.rs

```diff
@@ -73,14 +73,19 @@ async fn run() -> Result<()> {
 
     let entry_point = parse_entry_point(&args)?;
     let exit_point = parse_exit_point(&args)?;
+    let nym_ips = if let (Some(ipv4), Some(ipv6)) = (args.nym_ipv4, args.nym_ipv6) {
```

Comment:
optional: create a constructor for IPPair

Reply:
Done: https://github.com/nymtech/nym/pull/4440